### PR TITLE
Modified css and html

### PR DESCRIPTION
**Pull request by spaceTony9**
1. The project was updated with new prettier settings.
2. html: 
- Added wider description to all alt attributes.
- None of the img tags have width.
- To form tag added attributes: action and method.
- Several class names were reviewed and renamed.
3. css:
- to main css file was imported style-variables.ccs that contains variables needed for the project, particularly: color, shadows, transitions and flex-basis variables. 
- styles of the mobile menu were also separated to an independent file.
- main css file was edited, all repetitive rules were deleted, added structure to every css rule, variables were used. 
- Adaptive markup presented only with 2 media requests, all unnecessary requests were deleted.
4. File README.MD was edited.

### CODE REVIEW

File: css/style-variables.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    --navy-blue: #2E2F42;
+    --green: #31D0AA;
+    --slate: #434455;
+    --light-slate: #8E8F99;
+    --cornflower: #E7E9FC;
+    --cloud: #F4F4FD;
+    --navy-blue-modal: #2e2f4266;
+    --grey: #2e2f42b3;
+    --dairy: #FCFCFC;
+
```
Comment (Fernandovi4) [inline css/style-variables.css]: It is a good practice to apply alphabet sorting in such cases

File: css/styles-mobile-menu.css [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    z-index: 1;
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    height: 100%;
+    padding: 24px 24px 40px 16px;
+    background: var(--white);
+    overflow: auto;
+    opacity: 0;
+    visibility: hidden;
```
Comment (Fernandovi4) [inline css/styles-mobile-menu.css]: What is the purpose of usage both 'opacity' and 'visibility'?

File: css/styles.css [hunk 3]
```diff
@@ -48,289 +57,115 @@ img {
...
-}
-
-.container-values {
-}
-
-.section-values-header {
-    
-}
-
 .visually-hidden {
```
Comment (Fernandovi4) [inline css/styles.css]: What is the purpose of this class (and element itself)?

File: css/styles-mobile-menu.css (hunk 2 above)
Comment (spaceTony9) [reply css/styles-mobile-menu.css]: The visibility declaration used to hide the element without changing the layout of the page. Opacity used to make the element transparent for now, but the idea is that with the help of Javascript the opacity value would be changed so to make the block element sorts of appear smoothly on the screen.

File: css/styles.css (hunk 3 above)
Comment (spaceTony9) [reply css/styles.css]: Class .visually-hidden hides the header of the value section as per design there is no header. However, for the purpose of maintaining semantics on the page and ensuring correct work of search engines header was added. In perfect case, in header, the information that highlights the content of the section, be stated.

File: css/style-variables.css (hunk 1 above)
Comment (spaceTony9) [reply css/style-variables.css]: Thank you for the comment. I will definitely take this into account when working on variables file.

**Pull request closed without merging**